### ISSUE TITLE
Caldroid RefreshView

### ISSUE THREAD

**Issue by sohguanh** [roomorama/Caldroid#45]
Hi I am a new user of Caldroid. I don't know if this is a bug but just want to list down something that does not seem to allow refresh.

HashMap<Date, Integer> map = new HashMap<Date,Integer>();  
//put in values etc
m_fragCaldroid.setBackgroundResourceForDates(map);
m_fragCaldroid.refreshView();

Upon calling refreshView below error occurs

org.joda.time.IllegalFieldValueException: Value -1 for monthOfYear must be in the range [1,12]

Trace to below code in refreshView method
// Refresh title view
monthTitleTextView.setText(new DateTime(year, month, 1, 0, 0, 0, 0)
            .monthOfYear().getAsText().toUpperCase()
            + " " + year);

Basically I am calling setBackgroundResourceForDates for different map so I want to re-use the Caldroid fragment by doing a refresh every time I set a new map. Is this supported or whenever I change the map, I need to instantiate a new Caldroid fragment ?
